### PR TITLE
U/mattc/update docs

### PR DESCRIPTION
**Pull request by oholiab**
No code changes so no tests required to pass :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,8 +65,10 @@ best way to get a full view of the requisite state of the system for
 pidtree-bcc to work.
 
 ## Usage 
-> CAUTION! The Makefile calls 'docker run' with `--priveleged` so it is your
-> responsibility to ensure that it's not going to do anything untoward!
+> CAUTION! The Makefile calls 'docker run' with `--priveleged`,
+> `--cap-add=SYS_ADMIN` and `--pid host`` so it is your responsibility
```
Comment (piax93) [inline README.md]: Fix double backtick after `--pid host`

File: README.md [hunk 2]
```diff
@@ -93,6 +109,7 @@ making TCP ipv4 `connect` syscalls like this one of me connecting to Freenode in
   "timestamp": "2019-11-12T14:24:57.532744Z",
   "pid": 1775,
   "daddr": "185.30.166.37",
+  "saddr": "X.X.X.X",
```
Comment (smrojas) [inline README.md]: nit: not entirely true. Won't it just pull the IP from /etc/hosts, which _may_ contain ipv6 address?

:meh:

File: README.md [hunk 3]
```diff
@@ -114,3 +131,59 @@ ports:
...
+To enable the `sourceipmap` plugin, simply include a `plugins` stanza in the config like so:
+
+```yaml
+plugins:
+  sourceipmap:
+    enabled: True
+    hostfiles:
+      - '/etc/array`
+      - '/etc/of`
+      - '/etc/hostfiles`
```
Comment (piax93) [inline README.md]: <pre>s/`/'</pre>

File: README.md [hunk 4]
```diff
@@ -114,3 +131,59 @@ ports:
...
+    attribute_key: "source_host"
```
Comment (piax93) [inline README.md]: Fix single/double quotes to be consistent across snippets

File: README.md [hunk 5]
```diff
@@ -114,3 +131,59 @@ ports:
...
+
+If you then volume mount the *directory* that this file is in (the contents of the file will not update if you bind mount it in directly) to a location like `/maps`, you can then use a configuration like:
+
+```yaml
+plugins:
+  sourceipmap:
+    enabled: True
+    hostfiles:
+      - '/maps/ipmapping.txt'
+    attribute_key: "source_container"
```
Comment (piax93) [inline README.md]: Same as above

File: README.md [hunk 6]
```diff
@@ -114,3 +131,59 @@ ports:
 ```
 
 To see *only* events for these ports.
+
+## Plugins
+Plugin configuration is populated using the `plugin` key at the top level of the configuration:
```
Comment (piax93) [inline README.md]: `s/plugin/plugins`

File: README.md (hunk 2 above)
Comment (oholiab) [reply README.md]: No it will never match v6 addresses because saddr only ever outputs v4 ones so they'll get ignored

**Pull request merged**